spec/api/v2/identity/sessions_spec.rb

```diff
@@ -189,6 +189,20 @@
         end
       end
     end
+
+    context 'event API behavior' do
+      before do
+        allow(EventAPI).to receive(:notify)
+      end
+
+      it 'receive system.session.create notify' do
+        post uri, params: { email: email, password: password }
+
+        expect(EventAPI).to have_received(:notify).with('system.session.create',
+          hash_including({ record: hash_including(user: anything, user_ip: anything, user_agent: anything) })
```

Comment:
Could we also test the logic?
Maybe we mock `user_agent` and `user_ip` and ensure, that it returns right stuff.